source/initial-dns-seedlist-discovery/tests/README.rst

```diff
@@ -56,8 +41,11 @@ These YAML and JSON files contain the following fields:
 - ``uri``: a mongodb+srv connection string
 - ``seeds``: the expected set of initial seeds discovered from the SRV record
 - ``hosts``: the discovered topology's list of hosts once SDAM completes a scan
+- ``options``: the parsed connection string options as discovered from URI and
```

Comment:
Is it assumed that the `options` are grouped into one string and provided by a single TXT lookup? How do we test behavior for combining multiple returned TXT records?

Reply:
There is a test for one TXT record: https://github.com/mongodb/specifications/pull/194/files#diff-f2bf555cce5b896c1bb77ba6849a0c4f ; and one with two TXT records providing two partial strings: https://github.com/mongodb/specifications/pull/194/files#diff-a4ba8018388fff90fc88668ef545a68d

Reply:
What I was getting at is that there aren't any tests here that show the options being built from multiple TXT records, these are just testing once the records have been merged into a single connection string (it looks like).  We can add tests later, but I imagine this would be something we'd like to unit test in drivers